packages/p2p/src/stores/my-ads-store.js

```diff
@@ -22,24 +22,28 @@ export default class MyAdsStore extends BaseStore {
     current_method = { key: null, is_deleted: false };
     delete_error_message = '';
     edit_ad_form_error = '';
+    error_code = '';
```

Comment:
all of these new variables was pulled from my other PR which i needed to merge into this PR